_posts/pl/2022-01-11-smarthome.md

```diff
@@ -0,0 +1,316 @@
+W kwestii modyfikacji mogę dodać, że pierwszy czujnik po takiej zmianie podziałał 2 dni, po czym z nieznanych przyczyn przestał.
+Być może przegrzałem go przy lutowaniu, a być może po prostu „nastał jego czas”. Link do modyfikacji: [https://community.smartthings.com/t/making-xiaomi-motion-sensor-a-super-motion-sensor/139806](https://community.smartthings.com/t/making-xiaomi-motion-sensor-a-super-motion-sensor/139806)
+
+![Czujnik ruchu](/assets/img/posts/2022-01-11-smarthome/motionsensor.png){: width="250" }
+
+
+#### Sensory otwartych okien/drzwi
+
+Podobnie jak w przypadku czujników ruchu, tak i tutaj używam czujników tylko jednego rodzaju. 
+Szukałem jak najmniejszych, żeby nie rzucały się w oczy i po analizie kilku wybór padł na te od Aqara.
+Ciężko powiedzieć o nich coś więcej niż to, że dobrze spełniają swoją funkcję. 
+Zasięg między magnesem a czujnikiem może się wahać, w zależności czy urządzenia skierowane są do siebie frontami, czy bokiem.
+Mam urządzenia umieszczone w obydwu wariantach. odległość między nimi zazwyczaj wynosi około 1 cm i nie zaobserwowałem jeszcze żadnych problemów.
+
+![Głowica termostatyczna](/assets/img/posts/2022-01-11-smarthome/windowsensor.png){: width="250" }
+
+#### Głowice termostatyczne
+
+Temat głowic termostatycznych może być nieco bardziej zawiły, bo dostępnych jest sporo wariantów. 
+Ja wybrałem [Moes BRT-100-TRV](https://www.zigbee2mqtt.io/devices/BRT-100-TRV.html). Przy wyborze kierowałem się liczbą wystawianych informacji oraz zapewnieniem producenta, 
+że działają na baterii dłużej niż inne warianty.
+Szczegółowe informacje o wystawianych danych dostępne są na stronach projektu [Zigbee2MQTT](https://www.zigbee2mqtt.io/supported-devices/).
+Jedna z moich głowic raportuje stan baterii 91%, druga 83%. Obydwie działają od listopada, a w momencie sprawdzania tego stanu mieliśmy połowę stycznia.
+W kwestii baterii mogę dodać tylko, że ze względu na różnice w napięciu między bateriami a akumulatorami (1.2V vs 1.5V) konieczne może być używanie zwykłych baterii.
+
+![Głowica termostatyczna](/assets/img/posts/2022-01-11-smarthome/trv.png){: width="250" }
+
+#### Czujniki wycieku wody
+
+Konstrukcyjnie dosyć proste urządzenia. Na ich spodzie znajdują się dwie elektrody, których zwarcie powoduje zmianę stanu czujnika.
+W przypadku czujników Xiaomi/Aqara elektrody możemy wykręcić, zmniejszając tym samym poziom, od którego reagują.
+Najstarszy z moich czujników ma już dwa lata, a ciągle raportuje stan baterii na 100%, więc bez obaw można umieszczać je w trudno dostępnych miejscach.
+
+![Czujnik wycieku wody](/assets/img/posts/2022-01-11-smarthome/leaksensor.png){: width="250" }
+
+#### Czujniki temperatury
+
+Jedne z moich pierwszych czujników, a jak wspominałem, zaczynałem z bramką od Xiaomi, stąd wybór też tego producenta.
+Z zalet kwadratowych czujników temperatury Xiaomi mogę wymienić to, że poza temperaturą oraz wilgotnością raportują też ciśnienie atmosferyczne.
```

Comment:
Dużą zaletą kwadratowych czujników temperatury Xiaomi jest to, że poza temperaturą oraz wilgotnością, raportują też ciśnienie atmosferyczne.